mne/beamformer/_lcmv.py

```diff
@@ -285,6 +353,12 @@ def lcmv(evoked, forward, noise_cov, data_cov, reg=0.05, label=None,
         detected automatically. If int, the rank is specified for the MEG
         channels. A dictionary with entries 'eeg' and/or 'meg' can be used
         to specify the rank for each modality.
+    apply_noise_norm: False | True
+        If True, a noise-normalized beamformer will be computed.
```

Comment:
I still don't understand why you need both weight_norm and apply_noise_norm. Can you clarify this for me?